src/main/java/io/zold/api/Transaction.java

```diff
@@ -37,4 +36,32 @@
      * @return The amount
      */
     long amount();
```

Comment:
@t-izbassar as per the white paper, each Transaction has the following fields:

>1. `id` : Transaction ID, an unsigned 16-bit integer, 4-symbols hex;
>2. `time` : date and time, in ISO 8601 format, 20 symbols;
>3. `amount` : Zents, a signed 64-bit integer, 16-symbols hex;
>4. `prefix` : Payment prefix, 8-32 symbols;
>5. `bnf` : Wallet ID of the beneficiary, 16-symbols hex;
>6. `details` : Arbitrary text, matching `/[a-zA-Z0-9 -.]{1,512}/ ;`
>7. `signature` : RSA signature, 684 symbols in Base64.

Right now we only have `amount`. Let's add the other methods here too (you can leave a stub throwing `UnsupportedOperationException` in `Transaction.Smart`).